keyboards/m65/readme.md

```diff
@@ -0,0 +1,22 @@
+# M65
+
+![M65](https://i.imgur.com/gXuecush.jpg)
+
+A  (13x5) ortholinear keyboard that can be hand wired or using a pcb.
+
+* Keyboard Maintainer: [Alin Elena](https://gitlab.com/drFaustroll)
```

Comment:
This will link both your account here and your GitLab account as well.

```suggestion
* Keyboard Maintainer: [Alin Elena](https://github.com/alinelena) ([@drFaustroll on GitLab](https://gitlab.com/drFaustroll))
```